cocotb/share/makefiles/simulators/Makefile.xcelium

```diff
@@ -79,8 +79,8 @@ endif
 # which does not play nice with our multi language, so we supply GPI_EXTRA=cocotbvhpi if needed.
 
 #GPI_EXTRA:=
+GPI_ARGS = -loadvpi $(LIB_DIR)/libcocotbvpi:vlog_startup_routines_bootstrap
 ifeq ($(TOPLEVEL_LANG),verilog)
```

Comment:
Any idea why we need the vpi if the top-level language is VHDL?

Reply:
`vhpi` was and is loading via `GPI_EXTRA`. Would need to try other way.

Reply:
If one does ` -loadvpi $(LIB_DIR)/ius/libcocotbvhpi:vhpi_startup_routines_bootstrap`

```
     -.--ns INFO     cocotb.gpi                         ../embed/gpi_embed.c:71   in set_program_name_in_venv        Did not detect Python virtual environment. Using system-wide Python interpreter
     -.--ns INFO     cocotb.gpi                         ../gpi/GpiCommon.cpp:104  in gpi_print_registered_impl       VHPI registered
     -.--ns ERROR    cocotb.gpi                         ..vhpi/VhpiCbHdl.cpp:431  in arm_callback                    VHPI Error level 3: VHPI - Internal Error : Snapshot not found in Registering Callbacks !!
FILE (NULL):0
     -.--ns ERROR    cocotb.gpi                         ..vhpi/VhpiCbHdl.cpp:433  in arm_callback                    VHPI: Unable to register callback a handle for VHPI type vhpiCbStartOfSimulation(1034)
     -.--ns ERROR    cocotb.gpi                         ..vhpi/VhpiCbHdl.cpp:431  in arm_callback                    VHPI Error level 3: VHPI - Internal Error : Snapshot not found in Registering Callbacks !!
FILE (NULL):0
     -.--ns ERROR    cocotb.gpi                         ..vhpi/VhpiCbHdl.cpp:433  in arm_callback                    VHPI: Unable to register callback a handle for VHPI type vhpiCbEndOfSimulation(1035)

....

     -.--ns INFO     cocotb.gpi                         ../embed/gpi_embed.c:71   in set_program_name_in_venv        Did not detect Python virtual environment. Using system-wide Python interpreter
     -.--ns INFO     cocotb.gpi                         ../gpi/GpiCommon.cpp:104  in gpi_print_registered_impl       VHPI registered
     -.--ns ERROR    cocotb.gpi                         ..vhpi/VhpiCbHdl.cpp:439  in arm_callback                    VHPI ERROR: Registered callback isn't enabled! Got 1000

     -.--ns ERROR    cocotb.gpi                         ..vhpi/VhpiCbHdl.cpp:439  in arm_callback                    VHPI ERROR: Registered callback isn't enabled! Got 1000


```

Reply:
There are errors if one does different way.

Reply:
Needing VPI for VHDL is some specialty with Cadence apparently, there are some comments about this in the code IIRC.